### PR TITLE
Corrected some typos

### PR DESCRIPTION
**Pull request by criadoperez**
Corrected some typos

### CODE REVIEW

File: src/wycheproof/ecdsa_secp256k1_sha256_bitcoin_test.json
Comment (real-or-random) [inline src/wycheproof/ecdsa_secp256k1_sha256_bitcoin_test.json:1]: This file is a copy of https://github.com/google/wycheproof/blob/master/testvectors_v1/ecdsa_secp256k1_sha256_bitcoin_test.json . I don't think we want to touch it here. Please report upstream instead.

**Review by real-or-random — COMMENTED** [review]
Thanks!

ACK mod the wycheproof file

File: src/wycheproof/ecdsa_secp256k1_sha256_bitcoin_test.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "notes" : {
     "ArithmeticError" : {
       "bugType" : "EDGE_CASE",
-      "description" : "Some implementations of ECDSA have arithmetic errors that occur when intermediate results have extreme values. This test vector has been constructed to test such occurences.",
+      "description" : "Some implementations of ECDSA have arithmetic errors that occur when intermediate results have extreme values. This test vector has been constructed to test such occurrences.",
```
[the remark this answers is not in the document]
Comment (criadoperez) [reply src/wycheproof/ecdsa_secp256k1_sha256_bitcoin_test.json:13]: Thanks @sipa 
I have just submitted a [PR](https://github.com/google/wycheproof/pull/91) over there for this.

**Comment by criadoperez** [comment]
Removed the changes form the `ecdsa_secp256k1_sha256_bitcoin_test.json` file

**Comment by criadoperez** [comment]
Sure thing. No need to have multiple.

**Comment by real-or-random** [comment]
Oh, I think @hebasto was asking if *you* could squash the commits into a single one (and force-push your branch). Our usual workflow is that the PR author squashes, not the person merging.

**Comment by criadoperez** [comment]
Done. Squashed into one.

**Pull request merged**